pages/dataSummary.php

```diff
@@ -48,13 +49,25 @@
     }
     
     }
-
+    
+    /* overwriting some css of back to top */
+#button {
+  width: 42px;
+  height: 42px;
+  bottom: 10px;
+  right: -20px;
+}
+#button::after {
+  font-size: 1.5em;
+  line-height: 40px;
+  color: #fff;
+}
```

Comment:
is overwriting necessary? other pages which has back to the top button does not seem to have an additional css

Reply:
Original one is little bit big for this page it was overlapping with the 2nd table that's why i decrease its size a little bit so that it can fit perfectly